cognite_toolkit/examples/cdf_data_pipeline_asset_valhall/default.config.yaml

```diff
@@ -0,0 +1,33 @@
+# This is the configuration file used for the example data from The Open Industrial Data (oid)
+#
+# The data originates from a single compressor on Aker BP’s Valhall oil platform 
+# in the North Sea. Aker BP selected the first stage compressor on the Valhall 
+# because it is a subsystem with clearly defined boundaries, rich in time series and maintenance data.
+---
+# spesify the site/asset location where data comes from, ex 'valhall_oid' or if they are generic for all assets use 'all' 
+location_name: oid 
+module_version: '1'
+
+# spesify the name of the source making it possible to identify where the data orginates from, ex: 'workmate', 'sap', 'oracle',..
+source_name: workmate  
+
+asset_dataset: ds_asset_oid
+asset_raw_input_db: asset_oid_workmate
+asset_raw_input_table: assets
+
+# source ID from Azure AD for the corresponding groups, ex 'c74797ce-9191-4a4a-9186-8fe21c54c3de'
+asset_location_extractor_group_source_id: <update me>
+asset_location_processing_group_source_id: <update me>
+asset_location_read_group_source_id: <update me>
+
+
+# Transformation credentials
+clientId: ${IDP_CLIENT_ID}
```

Comment:
We should use the global cicd_ variables directly, not create new local ones. We have documented how to create local credentials per transformation.